### DISCUSSION TITLE
Roadmap Progress

### DISCUSSION THREAD

**Discussion by milanof-huma** [grpc/grpc-web#1259] [General]
Hello, I just wanted to ask if this project is active and there is a plan to progress on the roadmap https://github.com/grpc/grpc-web/blob/master/doc/roadmap.md i.e. **In-process Proxies**?

**Comment by sampajano** [comment]
Hi thanks for asking! This project is active, but there is no active plans on adding In-process Proxies support.

thanks!

**Comment by thatcort** [comment]
Hi, can you please be more specific? Which parts of the roadmap are currently in development? What is the expected timeline for the whole roadmap?

**Reply by sampajano** [reply]
Hi :) I'll aim to work on a roadmap update soon and provide an update here. Thanks!

UPDATE: ETA Early January

**Reply by glerchundi** [reply]
Thanks @sampajano

**Reply by sampajano** [reply]
Hi! A new update is provided at: https://github.com/grpc/grpc-web/pull/1326

Thanks!
